weblate/formats/models.py

```diff
@@ -24,6 +24,15 @@
 from weblate.utils.classloader import ClassLoader
 
 
+class ExporterLoader(ClassLoader):
+    def __init__(self):
+        super().__init__("WEBLATE_EXPORTERS", False)
+        self.errors = {}
```

Comment:
I don't think we need to collect the errors here. All exporters should import just fine. This is different with the formats because some of them need optional dependencies.